src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -54,19 +54,25 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public void Add(System.Net.CookieCollection cookies) { }
         public System.Collections.IEnumerator GetEnumerator() { return default(System.Collections.IEnumerator); }
         void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
+        public void CopyTo(Cookie[] array, int index) { }
+        public bool IsReadOnly { get; }
```

Comment:
Shouldn't the following members be changed from explicit to implicit to [match the netstandard 2.0](https://github.com/dotnet/standard/blob/28a81e4c2e46de9c858f4fed1ec3a296141d14de/netstandard/ref/System.cs#L4727-L4741)?:

From:

```c#
bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
object System.Collections.ICollection.SyncRoot { get { return default(object); } }
void System.Collections.ICollection.CopyTo(System.Array array, int index) { }
```

To:

```c#
public bool IsSynchronized { get { return default(bool); } }
public object SyncRoot { get { return default(object); } }
public void CopyTo(System.Array array, int index) { }
```

Also, I didn't see it mentioned in #12125, but shouldn't the following also be added to [match netstandard 2.0](https://github.com/dotnet/standard/blob/28a81e4c2e46de9c858f4fed1ec3a296141d14de/netstandard/ref/System.cs#L4733)?

```c#
public System.Net.Cookie this[int index] { get { return default(System.Net.Cookie); } }
```